pipelines/dynamodb/src/main/java/org/streamprocessor/pipelines/Dynamodb.java

```diff
@@ -164,7 +164,11 @@ public static void main(String[] args) {
         // options.setNumberOfWorkerHarnessThreads(10);
         options.setStreaming(true);
         options.setEnableStreamingEngine(true);
-        LOG.info("NumberOfWorkerHarnessThreads: " + options.getNumberOfWorkerHarnessThreads());
+        LOG.info(
```

Comment:
Maybe remove this log? Does it give us any useful information? Or we can keep it witht he TODO